css/toDoStyle.css

```diff
@@ -1,6 +1,104 @@
-*{
-    padding: 0;
-    margin: 0;
-    border-radius: 0;
 
-}
\ No newline at end of file
+body{
+    margin: 5% 6% 0% 5%
+
+}
+
+.headBar{
+    display: flex;
+    flex-direction: row;
+}
+h1{
+    flex-grow: 4;
+    color: rgba(72,16,114,255);
+    font-size: 7vw;
+
+}
+h2{
+    margin-top: -1%;
+    margin-bottom: -1%;
+    color: rgba(57,57,57,255);
+    font-style: normal;
+    font-size: 5vw;
+
+
+}
+.line{
+    width: 115%;
+    border-left: .1px solid black;
+    height: .00001vw;
+    display: inline-block;
+    box-shadow: 5px 5px 2px .5px  #888;
+    margin-left: -9%;
+  }
+#more{
+    transform: rotate(90deg); 
+    
+    
+}
+#searchIcon, #more{
+    -moz-box-sizing: border-box; 
+    -webkit-box-sizing: border-box; 
+    box-sizing: border-box; 
+    color: rgba(72,16,114,255);
+    background-color: rgba(246,243,248,255);
+    width: 35px;
+    height: 35px;
+    text-align: center;
+    border-radius: 50%;
+    padding-top: 9px;
+    margin-top: 4%;
+    margin-left: 1%;
+
+}
+#searchIcon{
+    margin-right: 2%;
+
+}
+
+#addTask{
+    color: white;
+    background-color: rgba(72,16,114,255);
+    border-radius: 50%;
+    position: fixed;
+    bottom: 40%;
+    right: 6%;
+    width: 56px;
+    height: 56px;
+    text-align: center;
+    line-height: 56px;  
+}
+
+#saveTask{
+    color: white;
+    background-color: rgba(72,16,114,255);
+    border-radius: 50%;
+    position: fixed;
+    bottom: 40%;
+    right: 6%;
+    width: 56px;
+    height: 56px;
+    text-align: center;
+    line-height: 56px;  
+    display: inline; 
+}
+
+#taskText{
+    width: 65%;
+    height: 80px;
+
+    border-radius: 2vw;
+    border: 1px solid rgba(237,237,237,255);
+    border-top: none;
+    border-color: rgb(223,223,223);
+    box-shadow: 0px 10px 20px -8px rgb(212, 211, 211);
+
+    margin-top: 50px;
+
+    padding-left: 5%;
+    padding-right: 20%;
+    
+    position: fixed;
+    bottom: 38.55%;
```

Comment:
maybe if we remove the outline it will look better 
![Screenshot from 2021-07-14 21-07-47](https://user-images.githubusercontent.com/47992412/125671592-c3b121e5-87ea-4fa2-85d3-2ea4bf67ed3a.png)

Reply:
![image](https://user-images.githubusercontent.com/82904792/125673637-5292fe4b-aff7-4447-bde8-59c15711e136.png)